api/src/main/java/com/capitalone/dashboard/auth/DefaultAuthenticationResponseService.java

```diff
@@ -64,6 +68,7 @@ public void handle(HttpServletResponse response, Authentication authentication)
 		
 		tokenAuthenticationService.addAuthentication(response, authenticationWithAuthorities);
 
+		busCompOwnerService.assignOwnerToDashboards(firstName, middleName, lastName, authentication);
```

Comment:
@Sbrenthughes Needs to move up inside the else and in an authType ldap if block.